### PR TITLE
General PIONEER conventions

### PR DESCRIPTION
**Pull request by EmV25**
@Anne0507 - feel free to add/adjust

### CODE REVIEW

File: conventions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# General conventions
+
+## OMOP CDM version
+PIONEER datasets are converted to [OMOP CDM V6.0.0](https://github.com/OHDSI/CommonDataModel/wiki) 
+plus [oncology extensions](https://github.com/OHDSI/OncologyWG/wiki).
+The *Episode* and *Episode Event* tables are added to the OMOP CDM. 
+In addition, the fields ``modifier_of_event`` and ``modifier_of_field_concept_id`` are added to the *Measurement* table.
+
+## OMOP Vocabulary version
+OMOP vocabulary v5 is used for the semantic mappings. The vocabularies are downloaded from [Athena](http://athena.ohdsi.org/).
```
[the remark this answers is not in the document]
Comment (EmV25) [reply conventions.md]: I will check and adjust this!

[the remark this answers is not in the document]
Comment (Anne0507) [reply conventions.md]: Voor PRIAS:  v5.0 26-FEB-20

[the remark this answers is not in the document]
Comment (EmV25) [reply conventions.md]: @MaximMoinat Can you check the vocabulary version of ERSPC?

[the remark this answers is not in the document]
Comment (MaximMoinat) [reply conventions.md]: `v5.0 01-AUG-19`, that is also the one used on the server

**Pull request merged**